### ISSUE TITLE
[med] JAWS+ Chrome: Role is not defined for the Save control.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8387]
## Action Performed:
1. Using JAWS+ Chrome, open URL staging.new.expensify.com
2. Navigate to already created room from the Chat list using Tab key and press Enter on it.
3. Select the Room name and open the Details section.
4. Navigate to Setting control using Tab key and press Enter to activate it.
5. Navigate to Save control using Tab key.
6. Observe the screen reader announcement.

## Expected Result:
Role = Button should be defined for the Save control.

## Actual Result:
Screen reader is reading only "Save" not reading the name of the control.

**Note-**
This issue repro on all pages where Save control is appearing.

**HTML Code-**
div class="css-1dbjc4n r-1777fci" style="background-color: rgb(3, 212, 124); border-radius: 8px; border-width: 0px; height: 52px; opacity: 1; padding: 8px 12px;"><div dir="auto" class="css-901oao r-633pao r-lrvibr" style="color: rgb(255, 255, 255); font-family: GTAmericaExp-Regular; font-size: 13px; font-weight: 700; line-height: 18px; text-align: center;" Save</div>/div>

**Other occurrences**
>Same issue repro on PR #8764, #8758, #8826, #8845, #8921, #8946, #9070
>Same issue repro on PR #8940, #8940, #9129, #9101, #9129 -Role is not defined for go back and forgot controls.
>Same issue repro on PR #8788 -Role is not defined for about and the controls present inside the expanded content.
>Same issue repro on PR #8876, #8752-Role is not defined for continue.

## Workaround:
Yes

## Area issue was found in:
Chat Room Settings

## Failed WCAG checkpoints
4.1.2

## User impact:
Screen reader users will not be made aware that these elements are buttons.

## Suggested resolution:
Set Role = 'Button' for Save control. 
http://a11y-guidelines.orange.com/mobile_EN/dev-ios.html#element-trait.

## Platform:
- Web
- Mobile Web

**Version Number:** v1.1.46-2
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

![8164_Role not defined for Save button](https://user-images.githubusercontent.com/97600950/161000058-e78b773d-131f-4279-b63f-7ab77ffab3e0.png

https://user-images.githubusercontent.com/97600950/161027278-c8b4351c-9728-4201-8a8e-733467796ce9.mp4

)
